### ISSUE TITLE
[NuGet.org Bug]: A11y_NuGet Service_Home_Create .NET apps faster with NuGet_Text Spacing: After applying text spacing 'Versions' text is not visible properly.

### ISSUE THREAD

**Issue by advay26** [NuGet/NuGetGallery#8988]
### Impact

It's more difficult to complete my work

### Describe the bug

Source: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1468954

### Repro Steps

1. Launch the application URL: https://www.nuget.org/
2. "Home" page will get open.
3. Apply text spacing and observe that 'Versions' text is visible properly or not.

After applying text spacing 'Versions' text is not visible properly.

Similar issue is observed for "Packages" text also.

### Expected Behavior

After applying text spacing all the content should be clearly visible.
